### ISSUE TITLE
Implementation varies a lot from original Deep and Confident Prediction for Time Series at Uber

### ISSUE THREAD

**Issue by reachsumit** [PawaritL/BayesianLSTM#3]
Was this intentional?

The original paper proposes somewhat different ideas than the current code in this repo, such as:

* Prediction network only uses encoder after the encoder-decoder structure is pre-trained.
* Encoder uses VariationalDropout while Prediction Network uses regular dropout.
* Inherent noise is modeled at inference time using a validation set.
* Encoder reads up to T timestamps and Decoder reconstructs next F timestamps in the window.
* Input is de-trended.

I'm only pointing out these differences because this repo shows up as the first implementation of the original paper on PapersWithCode website (link: https://paperswithcode.com/paper/deep-and-confident-prediction-for-time-series), but the implementation is very different from the idea in the paper.

**Comment by PawaritL** [comment]
Hi, sorry I wasn't even aware that my repo ended up on PaperWithCode. 
At the time, I was trying to create the easiest example. 
I've edited the markdown on the notebook now to clarify that it's not the same model as the Uber paper.
